### PR TITLE
Update Pbsrc.com.xml

### PR DESCRIPTION
**Pull request by terrorist96**
Fixes https://github.com/EFForg/https-everywhere/issues/3527

### CODE REVIEW

**Comment by terrorist96** [comment]
@J0WI I have no idea what I'm doing. It got pulled over here. I give up.

File: src/chrome/content/rules/Ars-Technica.xml [hunk 1]
```diff
@@ -100,5 +100,8 @@
 
 	<rule from="^http://(?:(?:media|origin)\.arstechnica\.com|(?:cdn|static)\.arstechnica\.net)/"
 		to="https://cdn.arstechnica.net/" />
+		
+	<rule from="^http://arstechnica.com/civis"
+		to="https://arstechnica.com/civis"
```
Comment (J0WI) [inline src/chrome/content/rules/Ars-Technica.xml]: Missing `/>`

Comment (terrorist96) [inline src/chrome/content/rules/Ars-Technica.xml]: Thanks. Fixed.

**Comment by terrorist96** [comment]
Would be great if this could be merged.

**Comment by J0WI** [comment]
This still doesn't pass our tests because of lacking test urls.

**Comment by terrorist96** [comment]
I'm not a pro. I don't know where to find test URLs. Can you please help?

**Comment by J0WI** [comment]
This contains some unrelated changes from Ars-Technica.xml.
I think without them you are just disabling a rule which doesn't require any tests.

Apart from that it would be great if you could just make a specific exclusion that we can still secure other sites that are not affected.

**Comment by terrorist96** [comment]
Idk how to make the exclusion specific. Can you please type out what I need to put in the commit? I made a new PR
https://github.com/EFForg/https-everywhere/pull/4522

**Pull request closed without merging**